src/NuGet.Core/NuGet.Protocol/HttpSource/HttpSourceAuthenticationHandler.cs

```diff
@@ -25,7 +25,9 @@ public class HttpSourceAuthenticationHandler : DelegatingHandler
         private readonly HttpClientHandler _clientHandler;
         private readonly ICredentialService _credentialService;
 
+#pragma warning disable CA2213
         private readonly SemaphoreSlim _httpClientLock = new SemaphoreSlim(1, 1);
+#pragma warning restore CA2213
```

Comment:
I would link the issue to the affected parts>

```suggestion
// https://github.com/NuGet/Home/issues/12116
#pragma warning restore CA2213
```